packages/edit-post/src/components/sidebar/post-template/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * WordPress dependencies
+ */
+import { useRef } from '@wordpress/element';
+import { PanelRow, Dropdown, Button } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import usePostTemplateForm from './hook';
+import PostTemplateForm from './form';
+
+export default function PostTemplate() {
+	const anchorRef = useRef();
+
+	const postTemplateForm = usePostTemplateForm();
```

Comment:
That's a really good point! I refactored this be a little more conventional (inline `useSelect` instead of `hook.js`) and in doing so made it so that `PostTemplate` loads the minimal amount of data necessary to render the label.